ci/cmake-single-platform.yml

```diff
@@ -1,4 +1,6 @@
-name: CMake
+# This starter workflow is for a CMake project running on a single platform. There is a different starter workflow if you need cross-platform coverage.
+# See: https://github.com/actions/starter-workflows/blob/main/ci/cmake-multi-platform.yml
+name: CMake on a single platform
```

Comment:
Along the same lines as the naming issue, maybe just `CMake on Ubuntu`?  There's a tradeoff:

* `CMake on Ubuntu` - The user gets a clear picture from the GitHub.com UI that the workflow is for Ubuntu
VS
* `CMake on a single platform` - The user won't need to make changes to the name if they change the nature of the workflow (or copy it to make a separate Windows or Mac workflow file). This also avoids the risk of having an "incorrect" name.

Reply:
Keeping the naming convention as is for now (see the other comment for the reasoning).